ignite/metrics/gan/fid.py

```diff
@@ -37,7 +37,7 @@ def fid_score(
     diff = mu1 - mu2
 
     # Product might be almost singular
-    covmean, _ = scipy.linalg.sqrtm(sigma1.mm(sigma2), disp=False)
+    covmean, _ = scipy.linalg.sqrtm(sigma1.mm(sigma2).detach().cpu().numpy(), disp=False)
```

Comment:
This is a bit strange that we need to call detach and twice cpu. Sigma1 and sigma2 are already on cpu. Maybe, we should detach on the lines 34, 35:
```python
    mu1, mu2 = mu1.detach().cpu(), mu2.detach().cpu()
    sigma1, sigma2 = sigma1.detach().cpu(), sigma2.detach().cpu()
```

Let's put this change into another PR, please

Reply:
https://github.com/pytorch/ignite/pull/3444